client/src/components/main/Landing/LandingPage.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Box, Typography, makeStyles, Link, Paper } from "@material-ui/core";
+import Carousel from "react-material-ui-carousel";
+
+const useStyles = makeStyles(() => ({
+  landingContent: {
+    width: "100%",
+    height: "100%",
+    marginTop: "15px",
+  },
+  carouselContainer: {
+    width: "100%",
+    height: "200px",
+    backgroundColor: "#2980b9",
+    color: "white",
+  },
+  carouselItem: {
+    marginLeft: "100px",
+    marginTop: "50px",
+  },
+  landingSection: {
+    fontSize: "20px",
+    margin: "15px",
+  },
+  fullSection: {
+    width:"100%", 
+    minHeight: "250px", 
+    marginTop: "50px"
+  },
+  splitSection: {
+    width:"49%", 
+    minHeight: "300px", 
+    marginTop: "30px"
+  },
+}));
+
+export default function LandingPage(): React.ReactElement {
+  const classes = useStyles()
+
+  return (
+    <Box className={classes.landingContent}>
+      <Paper elevation={0} className={classes.carouselContainer}>
```

Comment:
I'm not sure it should auto-rotate through the options -- looks like there's an `autoplay` bool you can set. Maybe we could also up the `interval` time.

Reply:
I'll take a look!

Reply:
updated!